### PR TITLE
fix: Adjust sell quantity when changing token

### PR DESCRIPTION
**Pull request by jorbuedo**
See comments in: https://emurgo.atlassian.net/browse/YOMO-899

### CODE REVIEW

File: packages/swap/src/translators/reactjs/state/state.ts [hunk 1]
```diff
@@ -87,7 +88,10 @@ export type SwapCreateOrderActions = Readonly<{
   limitPriceChanged: (limitPrice: Balance.Quantity) => void
   sellQuantityChanged: (quantity: Balance.Quantity) => void
   buyQuantityChanged: (quantity: Balance.Quantity) => void
-  sellTokenIdChanged: (tokenId: Balance.TokenInfo['id']) => void
+  sellTokenIdChanged: (
```
Comment (michalszorad) [inline packages/swap/src/translators/reactjs/state/state.ts]: Maybe `sellTokenChanged` to highlight the function does not take only id?

File: packages/swap/src/translators/reactjs/state/state.ts [hunk 2]
```diff
@@ -87,7 +88,10 @@ export type SwapCreateOrderActions = Readonly<{
...
+    tokenId: Balance.TokenInfo['id'],
+    denominationDifference: number,
+  ) => void
   buyTokenIdChanged: (tokenId: Balance.TokenInfo['id']) => void
```
Comment (michalszorad) [inline packages/swap/src/translators/reactjs/state/state.ts]: Do we need the same for `buyTokenIdChanged`?

Comment (jorbuedo) [reply packages/swap/src/translators/reactjs/state/state.ts]: No, the buy side recalculates based on the sell side, so the quantity is not kept.

File: packages/swap/src/translators/reactjs/state/state.ts (hunk 1 above)
Comment (jorbuedo) [reply packages/swap/src/translators/reactjs/state/state.ts]: https://github.com/Emurgo/yoroi/pull/2823 will probably implement this different

**Review by michalszorad — APPROVED** [review]

**Pull request closed without merging**